controllers/spacecleanup/space_cleanup_controller_test.go

```diff
@@ -104,6 +104,69 @@ func TestCleanupSpace(t *testing.T) {
 		assert.False(t, res.Requeue)
 	})
 
+	t.Run("with ParentSpace - Space shouldn't be deleted", func(t *testing.T) {
+		// given
+		parentSpace := spacetest.NewSpace("parentSpace",
+			spacetest.WithCreationTimestamp(time.Now().Add(-time.Minute)),
+		)
+		spaceBinding := spacebinding.NewSpaceBinding("johny", parentSpace.Name, "admin", "a-creator")
```

Comment:
```suggestion
		parentSpaceBinding := spacebinding.NewSpaceBinding("johny", parentSpace.Name, "admin", "a-creator")
```
so the name of this binding clearly refers to the parent space

Reply:
Changed in: [7eda771](https://github.com/codeready-toolchain/host-operator/pull/728/commits/7eda771d353d09901d0c50f121026a686e761946)